addons/binding/org.openhab.binding.zwave/ESH-INF/thing/_channels.xml

```diff
@@ -230,6 +230,16 @@
                 </options>
                 <limitToOptions>true</limitToOptions>
             </parameter>
+            <parameter name="config_invert_percent" type="boolean">
+                <label>Invert dimmer value</label>
+                <description>Invert the blinds dimmer value</description>
```

Comment:
I think we should remove the word 'dimmer' - it's not inverting a dimmer - it's inverting the percentage (which may, or no, be rendered as a dimmer widget)